client/public/0.1866668c66fc3e240ff4.hot-update.js

```diff
@@ -0,0 +1,176 @@
+webpackHotUpdate(0,{
```

Comment:
Same as above, if it is the whole folder, you can put the folder `public` in .gitignore